### ISSUE TITLE
Lexical error (in Text.ProtocolBuffers.Lexer): unknown character '/' (decimal 47)

### ISSUE THREAD

**Issue by alexbiehl** [k-bx/protocol-buffers#6]
Hey, 

I am trying to generate some stubs for some proto files. They start like this: 

```
/**
 * Licensed to the Apache Software Foundation (ASF) under one
 * or more contributor license agreements.  See the NOTICE file
 * 
 ... 
*/
```

Invoking `hprotoc` gives me 
```
hprotoc: user error (Parsing proto:
"../someproto.proto"
has failed with message
"someproto.proto" (line 1, column 0):
unexpected L_Error 1 "Lexical error (in Text.ProtocolBuffers.Lexer): unknown character '/' (decimal 47), at character 0 line 1 column 1."
expecting end of input, character ';', name "import", name "package", name "option", name "message", name "enum", name "extend" or name "service"
```

It seems `hprotoc` cannot handle comments well. 

A look in `Lexer.x` reveals, there is indeed a rule for comments: 

```
@inComment = ([^\*] | $white)+ | ([\*]+ [^\/])
@comment = [\/] [\*] (@inComment)* [\*]+ [\/] | "//".* | "#".*

... 
:-
...
  @comment; 
... 
```

I cannot figure out what is wrong here, as the comment expression looks ok. Any idea on that? 

(If I remove the comments everything is ok).

**Comment by alexbiehl** [comment]
Ah, I just realized it's not the `/** ... */` comments it seems to be the line `//` comments. Investigating now.

**Comment by k-bx** [comment]
Released under 2.1.6 version
